terms.html

```diff
@@ -22,47 +22,48 @@
+<p><span class="section-number">9</span>Unless we are prevented from doing so by law, we will notify you within 14 days of any requests by third parties relating to your use of PDF Tables (such as copyright infringement notices) or of any legal request for a third party to have access to your <em>content</em>.</p>
+<p><span class="section-number">10</span>We will not inform third parties about your use of PDF Tables unless we are required to do so by law or you have given us permission to do so.  We may monitor PDF Tables for duplication of activity by you and another user or users and offer to help you and them to combine resources in order to reduce your costs.  We shall not reveal your identity or the nature of your activities in these circumstances without having first obtained specific consent from you.</p>
 <p><span class="section-number">11</span>We will ensure that you have the means to extract your <em>content</em> from the system, so far as it is reasonable for us to do so in the circumstances.</p>
 <h2>Your commitment to us</h2>
-<p><span class="section-number">12</span> Your access to <em>the platform</em> is conditional upon you keeping to the terms of <em>this agreement</em>, including the Acceptable Use Policy.</p>
-<p><span class="section-number">13</span> You are solely responsible for your own <em>content</em> and your use of <em>the platform</em>, and you will compensate us for any losses we suffer as a result of hosting that <em>content</em> or your use of <em>the platform</em>, such as claims in copyright or relating to unauthorised access to third party web sites.</p>
+<p><span class="section-number">12</span> Your access to PDF Tables is conditional upon you keeping to the terms of <em>this agreement</em>, including the Acceptable Use Policy.</p>
+<p><span class="section-number">13</span> You are solely responsible for your own <em>content</em> and your use of PDF Tables, and you will compensate us for any losses we suffer as a result of hosting that <em>content</em> or your use of PDF Tables, such as claims in copyright.</p>
 <p><span class="section-number">14</span> You will ensure that your contact details and email address are kept accurate and up to date.</p>
 <p><span class="section-number">15</span> You consent to our sending you email at the email address you have supplied for purposes directly related to our service (for example announcements about service availability). We also offer an opt-in newsletter by email, which you may choose on sign up and opt into or out of at any time. </p>
```

Comment:
We also send emails (using Intercom) to registered users, and intend to do more. Two examples:

1) Currently we send feature announcements occasionally.
2) We intend to do drip marketing - so while somebody is onboarding, to send them tutorials and advice as they get to use the product.

They have an unsubscribe option in the emails to unsubscribe from all Internet emails.

Do we need opt-in to do those two things?

We also don't have an opt-in newsletter on PDFTables.com right now (although of course could add it).